src/api/c/var.cpp

```diff
@@ -177,7 +177,7 @@ af_err af_var(af_array* out, const af_array in, const bool isbiased,
 
         af_array no_weights = 0;
         af_var_bias bias =
-            (isbiased) ? AF_VARIANCE_POPULATION : AF_VARIANCE_SAMPLE;
+            (isbiased) ? AF_VARIANCE_SAMPLE: AF_VARIANCE_POPULATION;
```

Comment:
add any warning/note in the af.h header ?

Reply:
I think this sort of info should go into https://github.com/arrayfire/arrayfire/blob/master/docs/pages/release_notes.md and it will when we make a release.